src/components/m-dropdown/_preview.html

```diff
@@ -1,22 +1,5 @@
 <div style="min-height:200px;">
-  <axa-dropdown title="title" items='[{"name": "Item 1", "url": "#"}, {"name": "Item 2 loooooong", "url": "#"}, {"name": "Item 3", "url": "#"}]'>
-  </axa-dropdown>
+  
```

Comment:
Why are all example gone now?

Reply:
I removed all dropdown changes. they were not meant to be in this PR.

Reply:
something went wrong during revert, I still see this dropdowns as being removed

Reply:
There are no changes on dropdowns anymore.